benchmark/type/fold_right/brigand.list.cpp.erb

```diff
@@ -0,0 +1,23 @@
+// Copyright Bruno Dutra 2016
```

Comment:
ah yes copyright ;) I don't really care who gets the credit but probably should be consistant